emscripten.py

```diff
@@ -671,21 +666,12 @@ def create_sending(invoke_funcs, metadata):
   # Map of wasm imports to mangled/external/JS names
   send_items_map = {}
 
-  def add_send_items(name, mangled_name, ignore_dups=False):
-    # Sanity check that the names of emJsFuncs, declares, and globalImports don't overlap
-    if not ignore_dups and name in send_items_map:
-      assert name not in send_items_map, 'duplicate symbol in exports: %s' % name
```

Comment:
Is this assertion not important any more?

Reply:
Yeah this was more about sanity checking the output of wasm-emscripten-finalize.   Now that we do all the extraction locally its less important to assert there.